%YAML:1.2
# SPDX-FileCopyrightText: 2021 Smart Robotics Lab, Imperial College London, Technical University of Munich
# SPDX-FileCopyrightText: 2024 Smart Robotics Lab, Technical University of Munich
# SPDX-License-Identifier: CC0-1.0

map:
  dim:                        [15, 15, 15]
  res:                        0.01

data:
  # tsdf
  truncation_boundary_factor: 8
  max_weight:                 100

sensor:
  width:                      1200
  height:                     680
  fx:                         600.0
  fy:                         600.0
  cx:                         599.5
  cy:                         339.5
  near_plane:                 0.2
  far_plane:                  30.0

reader:
  reader_type:                "replica"
  sequence_path:              "/home/xiaoang/MyLab/replica_v1/room_0"
  ground_truth_file:          "/home/xiaoang/MyLab/replica_v1/room_0/traj_matrix.txt"
  lpips_model_path:           "/home/xiaoang/MyLab/GSFusion/parameter/vgg16.pth"
  fps:                        0.0
  drop_frames:                false
  verbose:                    0

app:
  enable_ground_truth:        true
  optim_params_path:          "/home/xiaoang/MyLab/GSFusion/parameter/optimization_params_replica.json"
  ply_path:                   "/home/xiaoang/MyLab/replica_v1/room_0/point_cloud"
  mesh_path:                  "/home/xiaoang/MyLab/replica_v1/room_0/mesh"
  slice_path:                 ""
  structure_path:             ""
  integration_rate:           1
  rendering_rate:             1
  meshing_rate:               0
  max_frames:                 600
  log_file:                   "/tmp/log.tsv"